lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -21,7 +22,12 @@ function restrictPropCoreFunctions()
         "propManipulate(e:vannn)"
         --"propBreak(e:)"
     }
+    
+    local globalrestrictedFunctions = {
```

Comment:
You could rename this to something that gives better understanding such as "adminOnlyFunctions"